src/meta/src/barrier/rpc.rs

```diff
@@ -526,7 +526,13 @@ impl ControlStreamManager {
                     (job_id, (definition.clone(), stream_job_fragments))
                 })
         });
-        let tracker = CreateMviewProgressTracker::recover(background_mviews, hummock_version_stats);
+        // FIXME(kwannoel): recover this for background ddl
+        let recovered_backfill_order_map = HashMap::new();
```

Comment:
What will happen if the creating MV with backfill order recovers with an empty state?

Reply:
IIUC, it will be treated as if without any backfill order.